perun-base/src/main/java/cz/metacentrum/perun/audit/events/UserManagerEvents/UserSyncFinishedWithErrors.java

```diff
@@ -0,0 +1,34 @@
+package cz.metacentrum.perun.audit.events.UserManagerEvents;
+
+import cz.metacentrum.perun.audit.events.AuditEvent;
+import cz.metacentrum.perun.audit.events.EngineIgnoreEvent;
+import cz.metacentrum.perun.core.api.User;
+
+public class UserSyncFinishedWithErrors extends AuditEvent implements EngineIgnoreEvent {
+
+	private User user;
+	private String message;
+
+	@SuppressWarnings("unused") // used by jackson mapper
+	public UserSyncFinishedWithErrors() {
+	}
+
+	public UserSyncFinishedWithErrors(User user) {
+		this.user = user;
+		this.message = formatMessage( "%s synchronization failed.", user);
```

Comment:
Is there right message? Because this one is same like for UserSyncFailed.java event.

Reply:
Copy paste bug, fixed